### PR TITLE
Port to turbo repo

### PR DESCRIPTION
**Pull request by jnutter**
Move to [Turbo](https://turborepo.org/) but keep lerna for package versioning and publishing.

Question: Should we not commit build artifacts?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,19 +6,41 @@
   "license": "MIT",
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build": "lerna run libbuild --stream",
-    "build-storybook": "lerna run libbuild --stream --scope=@cmpsr/components && lerna run build-storybook --stream",
-    "test": "lerna run test --stream",
-    "test:watch": "lerna run --parallel test:watch --stream",
-    "test:coverage": "lerna run --parallel test:coverage --stream",
+    "build": "turbo run libbuild",
```
Comment (tkanzakic) [inline package.json:9]: can we parallelise this?

File: package.json [hunk 2]
```diff
@@ -6,19 +6,41 @@
...
+    "build-storybook": "turbo run build-storybook",
+    "test": "turbo run test",
```
Comment (tkanzakic) [inline package.json:11]: ditto

File: package.json [hunk 3]
```diff
@@ -6,19 +6,41 @@
...
+    "test:watch": "turbo run test:watch --parallel",
+    "test:coverage": "turbo run test:coverage",
     "lint": "eslint . --ext .tsx,.ts,.js",
     "lint-and-fix": "eslint . --ext .tsx,.ts,.js --fix",
-    "publish-ci": "lerna run libbuild --stream && lerna publish from-package -y --no-verify-access"
+    "publish-ci": "turbo run libbuild && lerna publish from-package -y --no-verify-access"
```
Comment (tkanzakic) [inline package.json:16]: do you have an idea about how feasible is moving the release process from learna to turbo?

**Review by tkanzakic — APPROVED** [review]
LGTM!, just left a couple of comments about being able to parallelising some workflows.

File: package.json (hunk 3 above)
Comment (jnutter) [reply package.json:16]: turbo is not meant to be a release tool. We could do this in another tool however.

File: package.json (hunk 1 above)
Comment (jnutter) [reply package.json:9]: I dont think this is the best solution since it skips dependencies. Turbo tries to parallel the tasks as much as possible. https://turborepo.org/docs/reference/command-line-reference#--parallel

File: package.json (hunk 3 above)
Comment (jnutter) [reply package.json:16]: from their docs

> tl;dr you can use Lerna and Turbo together.

> Use lerna symlinking (lerna bootstrap) packages, publishing and changelog generation
> Use turbo for task running and caching
